cmd/vsphere-xcopy-volume-populator/e2e-tests/ansible/setup-vm.yml

```
@@ -0,0 +1,226 @@
+    vm_template: "{{ lookup('env', 'VM_TEMPLATE') | default('') }}"
+    vm_iso_path: "{{ lookup('env', 'VM_ISO_PATH') | default('') }}"
+    vm_os_type: "{{ lookup('env', 'VM_OS_TYPE') | default('linux-rhel8') }}"
+    vm_disk_size: "{{ lookup('env', 'VM_DISK_SIZE') | default('20') | int }}"
+    vm_disk_type: "{{ lookup('env', 'VM_DISK_TYPE') | default('thin') }}"
+    vm_memory: "{{ lookup('env', 'VM_MEMORY') | default('4096') | int }}"
+    vm_cpu: "{{ lookup('env', 'VM_CPU') | default('2') | int }}"
+    vm_test_data_size_mb: "{{ lookup('env', 'VM_TEST_DATA_SIZE_MB') | default('1024') | int }}"
+
+    # OS configurations
+    os_configs:
+      linux-rhel8: { guest_id: "rhel8_64Guest", is_linux: true }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)